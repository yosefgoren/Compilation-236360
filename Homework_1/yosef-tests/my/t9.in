// string with CR test:
 "there is a 
 here" 