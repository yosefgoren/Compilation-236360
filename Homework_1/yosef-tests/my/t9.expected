1 COMMENT //
Error unclosed string
